[5.1] Adding the missing seeOptionIsSelected and seeCheckboxIsChecked methods

sileence opened the pull request:
These methods will allow developers to test selects, radio groups and checkboxes, along with seeInField that allow developers to test input texts and textareas.

sileence committed:
Added the seeIsSelected and seeIsChecked methods for the integration test package

sileence commented:
I also added some integration tests this time:

https://github.com/sileence/framework/blob/check_field_values/tests/Foundation/FoundationCrawlerTraitIntegrationTest.php

taylorotwell commented:
Is there a strong reason not to just name them `seeIsSelected` and `seeIsChecked`?

sileence commented:
@taylorotwell I just decided to maintain the Codeception name convention. But if you consider the shorter names seeIsSelected and seeIsChecked also makes sense, just let me know, either way is fine.

taylorotwell commented:
To me the shorter names are better.

sileence commented:
@taylorotwell how would you abbreviate getSelectedValueFromSelect and getCheckedValueFromRadioGroup? Those are internal methods used by getSelectedValue, and they need to be specific.

sileence commented:
@taylorotwell done, I changed the names to seeIsSelected and seeIsChecked, I also renamed isCheckboxChecked to isChecked (I'm trying to separate the actual tests (methods prefixed by see*) from the helper methods that interact with the Crawler.

GrahamCampbell commented on src/Illuminate/Foundation/Testing/CrawlerTrait.php:
```diff
@@ -334,7 +367,101 @@ protected function getInputOrTextAreaValue($selector)
+    /**
+     * Get the checked value from a radio group.
+     *
+     * @param  Crawler  $radioGroup
```
use FQCN

sileence commented:
@GrahamCampbell fixed, comments now have FQCNs
